### PR TITLE
Linking dlgsummarybar to sdgLayeroptions

### PR DESCRIPTION
**Pull request by maxwellfundi**
@dannyparsons  you could have a look at this.

### CODE REVIEW

File: instat/dlgSummaryBarOrPieChart.vb [hunk 1]
```diff
@@ -184,7 +187,7 @@ Public Class dlgSummaryBarOrPieChart
     End Sub
 
     Private Sub cmdPieChartOptions_Click(sender As Object, e As EventArgs) Handles cmdPieChartOptions.Click
-        'sdgLayerOptions.SetupLayer(clsTempGgPlot:=clsRggplotFunction, clsTempGeomFunc:=clsRgeomBarFunction, clsTempAesFunc:=clsRaesFunction, bFixAes:=True, bFixGeom:=True, strDataframe:=ucrSummaryBarSelector.ucrAvailableDataFrames.cboAvailableDataFrames.Text, bApplyAesGlobally:=True)
+        sdgLayerOptions.SetupLayer(clsNewGgPlot:=clsRggplotFunction, clsNewGeomFunc:=clsRgeomBarFunction, clsNewGlobalAesFunc:=clsRaesFunction, clsNewLocalAes:=clsLocalRaesFunction, bFixGeom:=True, ucrNewBaseSelector:=ucrSummaryBarSelector, bApplyAesGlobally:=True, bReset:=bResetLayerSubdialog)
```
Comment (dannyparsons) [inline instat/dlgSummaryBarOrPieChart.vb]: You should have a reset boolean for each sub dialog, not the same one

Comment (maxwellfundi) [inline instat/dlgSummaryBarOrPieChart.vb]: So you mean that needs to be different?

Comment (dannyparsons) [inline instat/dlgSummaryBarOrPieChart.vb]: yes

Comment (maxwellfundi) [inline instat/dlgSummaryBarOrPieChart.vb]: Okay will change that for all dialogs then

Comment (dannyparsons) [inline instat/dlgSummaryBarOrPieChart.vb]: thanks

Comment (maxwellfundi) [inline instat/dlgSummaryBarOrPieChart.vb]: @dannyparsons  all this is now done

**Pull request merged**